.gitmodules

```diff
@@ -1,78 +1,3 @@
-[submodule "vendor-local/src/django-statsd"]
-	path = vendor-local/src/django-statsd
-	url = git://github.com/andymckay/django-statsd.git
-[submodule "vendor-local/src/pystatsd"]
-	path = vendor-local/src/pystatsd
-	url = git://github.com/jsocol/pystatsd.git
-[submodule "vendor-local/src/django-waffle"]
-	path = vendor-local/src/django-waffle
-	url = https://github.com/jsocol/django-waffle.git
-[submodule "vendor-local/src/rna"]
-	path = vendor-local/src/rna
-	url = git://github.com/mozilla/rna
-[submodule "vendor-local/src/django-extensions"]
-	path = vendor-local/src/django-extensions
-	url = https://github.com/django-extensions/django-extensions.git
-[submodule "vendor-local/src/django-rest-framework"]
-	path = vendor-local/src/django-rest-framework
-	url = https://github.com/tomchristie/django-rest-framework.git
-[submodule "vendor-local/src/requests"]
-	path = vendor-local/src/requests
-	url = https://github.com/kennethreitz/requests.git
-[submodule "vendor-local/src/Python-Markdown"]
-	path = vendor-local/src/Python-Markdown
-	url = https://github.com/waylan/Python-Markdown.git
-[submodule "vendor-local/src/jingo-markdown"]
-	path = vendor-local/src/jingo-markdown
-	url = https://github.com/monkeywarrior/jingo-markdown.git
-[submodule "vendor-local/src/six"]
-	path = vendor-local/src/six
-	url = https://github.com/kelp404/six.git
-[submodule "vendor-local/src/tweepy"]
-	path = vendor-local/src/tweepy
-	url = https://github.com/tweepy/tweepy.git
-[submodule "vendor-local/src/mdx_outline"]
-	path = vendor-local/src/mdx_outline
-	url = https://github.com/aleray/mdx_outline.git
 [submodule "vendor-local/src/legal-docs"]
 	path = vendor-local/src/legal-docs
-	url = https://github.com/mozilla/legal-docs.git
-[submodule "vendor-local/src/icalendar"]
-	path = vendor-local/src/icalendar
-	url = git://github.com/collective/icalendar.git
-[submodule "vendor-local/src/chkcrontab"]
-	path = vendor-local/src/chkcrontab
-	url = git://github.com/lyda/chkcrontab.git
-[submodule "vendor-local/src/django-pagedown"]
-	path = vendor-local/src/django-pagedown
-	url = https://github.com/timmyomahony/django-pagedown.git
-[submodule "vendor-local/src/django-filter"]
-	path = vendor-local/src/django-filter
-	url = https://github.com/alex/django-filter.git
-[submodule "vendor-local/src/django"]
-	path = vendor-local/src/django
-	url = git://github.com/django/django.git
-[submodule "vendor-local/src/bleach"]
-	path = vendor-local/src/bleach
-	url = https://github.com/jsocol/bleach.git
-[submodule "vendor-local/src/html5lib"]
-	path = vendor-local/src/html5lib
-	url = https://github.com/html5lib/html5lib-python
-[submodule "vendor-local/src/django-pipeline"]
-	path = vendor-local/src/django-pipeline
-	url = git://github.com/cyberdelia/django-pipeline.git
-[submodule "vendor-local/src/whitenoise"]
-	path = vendor-local/src/whitenoise
-	url = git://github.com/evansd/whitenoise.git
+	url = git://github.com/mozilla/legal-docs.git
```

Comment:
Sure. I removed it accidentally and I guess added it back differently. I'll update.